spec/integration/api/sys/mount_spec.rb

```diff
@@ -22,6 +31,24 @@ module Vault
         expect(result.type).to eq("aws")
         expect(result.description).to eq("")
       end
+
+      it "returns the options from a mount" do
+        next unless has_options_for_mount?
+
+        expect(subject.mount("kv_v2_mount", "kv", 'kv v2 mount', options: {version: "2"})).to be(true)
```

Comment:
isn't this supposed to be 
```suggestion
        expect(subject.mount("kv_v2_mount", "kv", 'kv v2 mount', version: "2")).to be(true)
```

Reply:
Nope, it's an options hash containing the entry of version . AFAIK :https://www.vaultproject.io/api/system/mounts.html#options